용재/17413.py

```diff
@@ -0,0 +1,27 @@
+getText = input()
+
+result = []
+stack = []
+
+flag = False
+
+for each in getText:
+  if each == '>':
+    result.append(each)
+    flag = False
+  elif each == '<' or flag == True:
+    while stack:
+      result.append(stack.pop())
+    result.append(each)
+    flag = True
+
+  elif each == ' ':
+    while stack:
+      result.append(stack.pop())
+    result.append(' ')
```

Comment:
하드코딩 대신 `each` 변수명을 사용하는 것이 좋아보입니다!

```suggestion
  elif each == ' ':
    while stack:
      result.append(stack.pop())
    result.append(each)
```

Reply:
오 이부분 확실히 문제가 될 수 있겠네요. 짚어주셔서 감사합니다!